source/class/qx/ui/mobile/dialog/Popup.js

```diff
@@ -239,9 +239,6 @@ qx.Class.define("qx.ui.mobile.dialog.Popup", {
 
         this.__registerEventListener();
 
-        // Move outside of viewport
-        this.placeTo(-1000, -1000);
-
```

Comment:
I think it is fine to remove this. I assume it was put there by someone who thought that the screen would potentially be updated during the show() call, and wanted this to happen off screen as it had the potential of looking ugly or something ... but since the screen does not get updated while js code is running AFAIK this precaution is not necessary and this the code can be removed.